NuGet.config

```diff
@@ -6,6 +6,9 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-windowsdesktop -->
+    <add key="darc-pub-dotnet-windowsdesktop-4fa7338" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-windowsdesktop-4fa7338a/nuget/v3/index.json" />
```

Comment:
@dotnet/dotnet-windowsdesktop-admin Is this new package source addition intentional? It seems unlike any of the others, not sure how this got added and if we want it,

Reply:
This probably comes from the installer's release/8.0.2xx branch consuming the stable windowsdesktop release/8.0.1xx bits. It's unrelated to the VMR work.

Reply:
Thank you for confirming. :) Waiting for confirmation on the wpf team to see if that's the right thing to do

Reply:
I'm not aware of anything that went into the dotnet-windowsdesktop release/8.0 branch that would cause this addition. I looked at some nuget.configs from other releases and they also have some additional package sources for windowdesktop, so I assume that this is coming from what Viktor is mentioning.

Reply:
Thank you for taking a look. Im guessing we dont want the 8.0.200 one to use 8.0.100 anymore, so Ill try to remove it.